snippets/matter-debug/matter_debug.conf

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable OpenThread debug logs
+CONFIG_OPENTHREAD_SOURCES=y
+CONFIG_OPENTHREAD_DEBUG=y
+CONFIG_OPENTHREAD_LOG_LEVEL_INFO=y
+CONFIG_OPENTHREAD_SHELL=y
+
+# Deferred mode of logging
+CONFIG_LOG=y
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_BUFFER_SIZE=40960
```

Comment:
Does it fit with this value on Light Bulb e.g. on nRF54L or we need to decrease it a bit?